frontend/packages/dev-console/src/components/monitoring/alerts/monitoring-silences.tsx

```diff
@@ -50,7 +50,7 @@ const ConnectedPage: React.FC<{ children: React.ReactNode }> = ({ children }) =>
   return <>{children}</>;
 };
 
-export const MonitoringEditSilencePage: React.FC<{ match: any }> = ({ match }) => (
+export const MonitoringEditSilencePage: React.FC<{ match: any }> = () => (
```

Comment:
Looks like we can remove the Component type definition as we are not using the match anymore.